aspnetcore/blazor/hybrid/tutorials/maui.md

```diff
@@ -114,36 +122,163 @@ Close the **Android Device Manager** window. Wait until the emulated phone windo
+
+:::zone-end
+
+:::zone pivot="macos"
+
+This tutorial shows you how to build and run a .NET MAUI Blazor app. You learn how to:
+
+> [!div class="checklist"]
+> * Create a .NET MAUI Blazor app project in Visual Studio for Mac
+> * Run the app on macOS with Mac Catalyst
+> * Run the app on an emulated mobile device in the Android Emulator
+
+## Prerequisites
+
+* [Supported platforms (.NET MAUI documentation)](/dotnet/maui/supported-platforms)
+* [Apple ID](https://appleid.apple.com/account) / [Apple Developer Program](https://developer.apple.com/programs/)
+* [Apple Xcode](https://developer.apple.com/xcode/)
+* [Visual Studio for Mac](https://visualstudio.microsoft.com/vs/mac/) with the **.NET Multi-platform App UI development** workload.
+
+> [!IMPORTANT]
+> Install Apple Xcode before installing Visual Studio for Mac.
+>
+> If the IDEs are installed in reverse order, set the Apple SDK in Visual Studio for Mac with the following steps:
+>
+> 1. Select **Visual Studio** > **Preferences** from the menu bar in Visual Studio for Mac.
+> 1. Under **SDK Locations** > **Apple**, set the **Apple SDK** > **Location** path to the physical location of the Xcode app.
+> 1. Select the **Restart Visual Studio** button.
+
+For more information on prerequisites and installing software on macOS for this tutorial, see the following resources in the .NET MAUI documentation:
+
+* [Supported platforms for .NET MAUI apps](/dotnet/maui/supported-platforms)
+* [Installation (Visual Studio for Mac)](/dotnet/maui/get-started/installation?tabs=vsmac)
+
+## Create a .NET MAUI Blazor app
+
+Launch Visual Studio for Mac. If the **Start Window** isn't open, select **Show Start Window** from the **File** menu.
+
+Select the **New** button to create a new app:
+
+:::image type="content" source="maui/_static/mac/step01.png" alt-text="Visual Studio for Mac Start .":::
+
+Select the **.NET MAUI Blazor App** project template and select the **Continue** button:
+
+:::image type="content" source="maui/_static/mac/step02.png" alt-text="Choose a template for your new project dialog with the .NET MAUI Blazor App project template selected.":::
+
+Select the target framework with the **Target framework** dropdown and select the **Continue** button:
+
+:::image type="content" source="maui/_static/mac/step03.png" alt-text="Configure your new .NET MAUI Blazor App dialog with the .NET 7.0 target framework selected.":::
+
+Name the project in the **Project name** field and select the **Create** button. The project name for this demonstration is `MauiBlazor`:
+
+:::image type="content" source="maui/_static/mac/step04.png" alt-text="Configure your new .NET MAUI Blazor App dialog with an project name of MauiBlazor.":::
+
+## Run the app on macOS with Mac Catalyst
+
+Select the play button (▶) to build and run the app:
+
+:::image type="content" source="maui/_static/mac/step05.png" alt-text="MauiBlazor window indicating the location of the play button.":::
```

Comment:
This screenshot shows a red squiggle under the component in the editor. Can we get a screenshot without that? Or is the Razor tooling in VS4Mac for .NET MAUI Blazor apps broken?

Reply:
***Confirmed!*** The red squiggle is still there in the UI. I'm opening a **Report a Problem** issue now from the MacBook. I'll post the link to it here when they send it.

As for the image here, I'll Photoshop that squiggle out 💪.

Reply:
Here's the Developer Community report on this one ...

https://developercommunity.visualstudio.com/t/Cant-find-SurveyPrompt-component-in-a-/10206835